src/data_availability.rs

```diff
@@ -489,16 +489,20 @@ pub mod tests {
             block: SignedBlock,
             tcp_server: &mut DaTcpServer,
         ) {
-            let full_block = self.node_state.handle_signed_block(&block);
+            _ = log_error!(
+                self.da.handle_signed_block(block.clone(), tcp_server).await,
+                "Handling Signed Block"
+            );
+            // TODO: we use this in autobahn_testing, but it'd be cleaner to separate it.
+            let Ok(full_block) = self.node_state.handle_signed_block(&block) else {
+                tracing::warn!("Error while handling signed block {}", block.hashed());
+                return;
+            };
```

Comment:
OPT: 
```suggestion
            let full_block = self.node_state.handle_signed_block(&block).context("Error while handling signed block")?;
```
jcrois que ça fait pareil (à check tho)

Reply:
C'est pour les tests je me prends pas la tête